src/test/java/com/artipie/rpm/meta/ChangelogEntryTest.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2020 artipie.com
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included
+ * in all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.artipie.rpm.meta;
+
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.core.IsEqual;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test for {@link ChangelogEntry}.
+ *
+ * @since 0.8.3
+ */
+class ChangelogEntryTest {
+
+    /**
+     * Entry tested.
+     */
+    private ChangelogEntry entry;
+
+    @BeforeEach
+    void setUp() {
+        this.entry = new ChangelogEntry(
+            "* Wed May 13 2020 John Doe <johndoe@artipie.org> - 0.1-2\n- Second artipie package"
```

Comment:
@olegmoz to make the tests more independant, I think it would be better not to share state/data between them even if it means duplicating the entry creation in each of them.

Reply:
@victornoel sorry, cannot agree with you here. I think that there is no reason to duplicate the string multiple times or invent some different inputs for each test

Reply:
@olegmoz see https://www.yegor256.com/2016/05/03/test-methods-must-share-nothing.html for a detailed rationale

Reply:
@victornoel thanks, I am familiar with this article and believe that it does more harm then good

Reply:
@olegmoz I am curious as to what do you find problematic with it?
@g4s8 can you decide what to do here?

Reply:
@victornoel @olegmoz I don't see any problem in this particular case, since we're testing one target against two different assertions. But I don't see the reason for `BeforeEach` in this class, I think it can be moved to private final field instead.

Reply:
@g4s8 @victornoel agree with this point, pushed an update